examples/grib_plot.py

```diff
@@ -3,26 +3,59 @@
 import pandas as pd
 import tempfile
 import pygrib
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+#formatter = logging.Formatter('%(asctime)s:%(levelname)s:%(name)s:%(message)s')
+
+# file_handler = logging.FileHandler('test2.log')
+# file_handler.setFormatter(formatter)
+
+# file_handler = logging.FileHandler('sample.log')
+# file_handler.setLevel(logging.ERROR)
+# file_handler.setFormatter(formatter)
+#logger.addHandler(file_handler)
 
 class GribPlot(bokeh_garden.application.AppWidget, bokeh.models.layouts.Column):
+
     def __init__(self, app, **kw):
         self._app = app
         self._sel = bokeh_garden.select.JSONSelect()
+        print("selection before")
         self._sel.on_change("value", self.sel_changed)
+        print("selection after")
+
+
+
+   #     self._test = bokeh_garden.progress_bar.ProgressBarTest(app)
+   #     self._bar = bokeh_garden.progress_bar.ProgressBar(app)
+
         self._source = bokeh.models.ColumnDataSource(data={"images": [], "x": [], "y": [], "dw": [], "dh":[]})
+        print("D source", self._source)
         self._figure = bokeh.plotting.Figure()
+        print("E figure", self._figure)
         self._figure.image("images", x="x", y="y", dw="dw", dh="dh", palette="Spectral11", source=self._source)
 
+
         bokeh.models.layouts.Column.__init__(self, self._sel, self._figure, **kw)
-        
-        self._app.on_change("grib", self.grib_changed)
```

Comment:
Please change this line back to how it used to be...

Reply:
done